src/components/Profile/Motto/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { Check, PencilAlt } from '@styled-icons/fa-solid'
+import React, { useEffect, useState } from 'react'
+import { useForm } from 'react-hook-form'
+import FormTextArea from '../../Form/components/FormTextArea'
+import { StyledParagraph } from '../../StyledText/StyledParagraph'
+import { Edit } from '../styled'
+
+type Props = {
+	motto: string | undefined
+}
+
+export const Motto: React.FC<Props> = ({ motto }) => {
+	const [element, setElement] = useState<boolean>(true)
+
+	const [text, setText] = useState<string | undefined>('')
+
+	useEffect(() => {
+		setText(motto)
+	}, [motto])
+
+	const { register, handleSubmit } = useForm()
+
+	const changeText = (text: string | undefined) => setText(text)
+
+	return (
+		<>
+			{element ? (
+				<form onSubmit={handleSubmit(() => setElement(false))}>
+					<FormTextArea
+						register={register}
+						value={text}
+						placeholder="Napiš své motto"
+						change={changeText}
+					/>
+					<Edit name="submit" type="submit">
+						<Check />
+					</Edit>
+				</form>
+			) : (
+				<>
+					<StyledParagraph>
+						{text?.length ? text : 'Motto není vyplněné'}
```

Comment:
In case of no motto, I'd add something like * * * (centered). Any text looks like we're forcing them to put some motto in while stars of some kind of "divider" works even without any text present.

Reply:
ok 👍